account/src/account/info.rs

```diff
@@ -0,0 +1,207 @@
+use std::collections::hash_map::DefaultHasher;
+use std::hash::{Hash, Hasher};
+use std::str::FromStr;
+use async_trait::async_trait;
+use base64::Engine;
+use novax::CodeMetadata;
+use num_bigint::BigUint;
+use novax_data::Address;
+use serde::{Deserialize, Serialize};
+use novax::caching::CachingStrategy;
+use novax::errors::NovaXError;
+use novax_request::gateway::client::GatewayClient;
+use crate::error::account::AccountError;
+use crate::utils::data::{code_metadata_deserialize, code_metadata_serialize};
+
+#[derive(Serialize, Deserialize, Default)]
+struct GatewayAccountInfo {
+    pub address: String,
+    pub nonce: u64,
+    pub balance: String,
+    pub username: String,
+    pub code: String,
+    #[serde(rename = "codeHash")]
+    pub code_hash: Option<String>,
+    #[serde(rename = "rootHash")]
+    pub root_hash: String,
+    #[serde(rename = "codeMetadata")]
+    pub code_metadata: Option<String>,
+    #[serde(rename = "developerReward")]
+    pub developer_reward: String,
+    #[serde(rename = "ownerAddress")]
+    pub owner_address: String,
+
+}
+
+#[derive(Serialize, Deserialize, Default)]
+struct GatewayBlockInfo {
+    pub nonce: u64,
+    pub hash: String,
+    #[serde(rename = "rootHash")]
+    pub root_hash: String
+
+}
+
+#[derive(Serialize, Deserialize, Default)]
+struct GatewayAccountInfoData {
+    pub account: GatewayAccountInfo,
+    #[serde(rename = "blockInfo")]
+    pub block_info: GatewayBlockInfo
+
+}
+
+#[derive(Serialize, Deserialize, Default)]
+struct GatewayAccount {
+    data: GatewayAccountInfoData
+}
+
+#[derive(Serialize, Deserialize, Default)]
+pub struct AccountInfo {
```

Comment:
The address field is missing. Despite this field is obvious it is a plus to have it in this struct too, so the developer can handle the data in a easier manner